spec/features/staff/users_can_create_a_fund_spec.rb

```diff
@@ -1,43 +1,72 @@
 RSpec.feature "Users can create a fund" do
-  before do
-    authenticate!(user: user)
-  end
-
-  let(:organisation) { create(:organisation, name: "UKSA") }
-  let(:user) { create(:user, organisations: [organisation]) }
-
   context "when the user is not logged in" do
     it "redirects the user to the root path" do
+      organisation = create(:organisation)
+      authenticate!(user: build_stubbed(:user))
```

Comment:
This reads strangely - why are we calling `authenticate` for a test case where the user is specifically *not* logged in?